CHANGES.rst

```diff
@@ -36,6 +36,8 @@ Changes:
 - Add reference link to ReadTheDocs URL of `Weaver` in API landing page.
 - Add references to `OGC-API Processes` requirements and recommendations for eventual conformance listing
   (relates to `#231 <https://github.com/crim-ca/weaver/issues/231>`_).
+- In order to align with OpenAPI `boolean` type definitions, non explicit `boolean` values will not be automatically 
+  converted to `bool` anymore. They will require explicit `false|true`.
```

Comment:
reword with
> Removed implicit ``boolean`` conversion from boolean-like values (``0``, ``1``, ``"true"``, etc.) and enforce explicit `true|false` values wherever applicable to align schema parsing with OpenAPI ``boolean`` type definitions.

Make sure to use double `` symbol for RST parsing to use code-style.